lib/mix/test/mix/dep_test.exs

```diff
@@ -248,15 +248,14 @@ defmodule Mix.DepTest do
     @behaviour Mix.RemoteConverger
 
     def remote?(_app), do: true
+    def deps(_dep, _lock), do: []
+    def post_converge, do: :ok
```

Comment:
Does it matter? They are stubbed out functions in a test module. I'd rather have them compact to reduce the noise for the rest of the tests.